onyo/cli/new.py

```diff
@@ -31,7 +31,11 @@
         metavar='TEMPLATE',
         required=False,
         help=r"""
-            Name of a template to populate the contents of new assets.
+            Path to a template to populate the contents of new assets.
+            If a relative path is given, it will first tried to be interpreted
```

Comment:
Nit: tried -> try

But I'd like to trim this down a bit. How about something like:

> If a relative path is given, it will first tried to be interpreted as relative to the template directory, allowing to simply give a name. Only if that doesn't exist will the path be resolved as relative to the current working directory.

to

> Relative paths will first attempt to resolve relative the to the template directory (i.e. just the template name), otherwise it will resolve relative to the current working directory.